entity-framework/core/modeling/owned-entities.md

```diff
@@ -16,184 +17,107 @@ EF Core allows you to model entity types that can only ever appear on navigation
+It is also possible to use the `OwnsOne` method in `OnModelCreating` to specify that the `ShippingAddress` property is an Owned Entity of the `Order` entity type and to configure additional facets if needed.
 
-``` csharp
-modelBuilder.Entity<Order>().OwnsOne(typeof(StreetAddress), "ShippingAddress");
-```
+[!code-csharp[OwnsOne](../../../samples/core/Modeling/OwnedEntities/OwnedEntityContext.cs?name=OwnsOne)]
 
-In this example, we use the `OwnedAttribute` to achieve the same goal:
+If the `ShippingAddress` property is private in the `Order` type, you can use the string version of the `OwnsOne` method:
 
-``` csharp
-[Owned]
-public class StreetAddress
-{
-    public string Street { get; set; }
-    public string City { get; set; }
-}
+[!code-csharp[OwnsOneString](../../../samples/core/Modeling/OwnedEntities/OwnedEntityContext.cs?name=OwnsOneString)]
 
-public class Order
-{
-    public int Id { get; set; }
-    public StreetAddress ShippingAddress { get; set; }
-}
-```
+See the [full sample project](https://github.com/aspnet/EntityFramework.Docs/tree/master/samples/core/Modeling/OwnedEntities) for more context. 
 
 ## Implicit keys
 
-In EF Core 2.0 and 2.1, only reference navigation properties can point to owned types. Collections of owned types are not supported. These reference owned types always have a one-to-one relationship with the owner, therefore they don't need their own key values. In the previous example, the StreetAddress type does not need to define a key property.  
+Owned types configured with `OwnsOne` or discovered through a reference navigation always have a one-to-one relationship with the owner, therefore they don't need their own key values as the foreign key values are unique. In the previous example, the `StreetAddress` type does not need to define a key property.  
 
-In order to understand how EF Core tracks these objects, it is useful to think that a primary key is created as a [shadow property](xref:core/modeling/shadow-properties) for the owned type. The value of the key of an instance of the owned type will be the same as the value of the key of the owner instance.      
+In order to understand how EF Core tracks these objects, it is useful to think that a primary key is created as a [shadow property](xref:core/modeling/shadow-properties) for the owned type. The value of the key of an instance of the owned type will be the same as the value of the key of the owner instance.
+
+## Collections of owned types
+
+>[!NOTE]
+> This feature is new in EF Core 2.2.
+
+To configure a collection of owned types `OwnsMany` should be used in `OnModelCreating`. However the primary key will not be configured by convention, so it need to be specified explicitly. It is common to use a complex key for these type of entities incorporating the foreign key to the owner and an additional unique property that can also be in shadow state:
```

Comment:
@AndriySvyryd I testing that OwnedAttribute works for owned collections if you specify a key member with the KeyAttribute. Can you confirm that this didn't work by accident? I think it is probably work mentioning.

Reply:
Filed https://github.com/aspnet/EntityFrameworkCore/issues/13874 and https://github.com/aspnet/EntityFrameworkCore/issues/13875.